Modules/IO/RAW/wrapping/itkRawImageIO.wrap

```diff
@@ -1,5 +1,8 @@
 itk_wrap_class("itk::RawImageIO" POINTER)
+  UNIQUE(to_types "UC;${WRAP_ITK_SCALAR}")
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
-    itk_wrap_template("${ITKM_F}${d}" "${ITKT_F},${d}")
+    foreach(t ${to_types})
+      itk_wrap_template("${ITKM_${t}${d}}${ITKM_${t}${d}}" "${ITKT_${t}${d}},${ITKT_${t}${d}}")
+    endforeach()
   endforeach()
```

Comment:
```suggestion
  itk_wrap_image_filter("${to_types}" 2)
```

Reply:
There seems to be something wrong with C++ template type specification. Let's try this shorter form.